react/introduction/fundamentals/react-what-is-state.md

```diff
@@ -47,9 +47,9 @@ class Hello extends React.Component {
 }
 ```
 
-This will throw an error, however, because `this.state` is `null` at the moment. The error would be `Cannot read property 'x' of null`.
+This will throw an error, however, because `this.state` is `undefined` at the moment. The error would be `Cannot read property 'x' of undefined`.
```

Comment:
hmm, seems like this is the error i get:
![image](https://user-images.githubusercontent.com/19802557/78037035-e984b880-7362-11ea-9f12-a3e163418008.png)

```suggestion
This will throw an error, however, because `this.state` is `undefined` at the moment. The error would be `TypeError: this.state is null`.
```